logstash/pipeline/medic.conf

```diff
@@ -2,7 +2,7 @@
 input {
     couchdb_changes {
         always_reconnect => true
-        db => "couchdb"
+        db => "medic"
```

Comment:
Isn't the purpose of `COUCHDB_DBS ` to define a list of databases you want to sync? Can we use that environment variable here? However, it may only work when it contains one database e.g "medic" and not a list e.g "medic medic_sentinel"

Reply:
@njuguna-n can you answer @lorerod's question above? I know you two had some conversations, and capturing the outcome for the public could benefit those who want to know more.

Reply:
This change is meant to update the default database that logstash listens to.  @lorerod your last sentence is true as well, to be able to target multiple databases you need to do that on separate `.conf` files for example `medic_sentinel` should have a `medic_sentinel.conf` file.